color_refinement.py

```diff
@@ -86,7 +88,68 @@ def color_refine(coloring: Coloring) -> Coloring:
     return coloring
 
 
-def get_number_isomorphisms(g: Graph, h: Graph, count: bool) -> int:
+def fast_color_refine(graph, coloring: "Coloring") -> "Coloring":
+    # Start with first color
+    qlist = DoubleLinkedList()
+    for c in sorted(coloring.colors):
```

Comment:
Moeten alle color classes in de queue of alle - 1?

Reply:
Geen idee, dus kunnen we ze er beter allemaal in gooien.